src/VisualStudio/Core/Def/Implementation/LanguageClient/VisualStudioLogHubLoggerFactory.cs

```diff
@@ -60,7 +60,7 @@ public async Task<ILspLogger> CreateLoggerAsync(string serverTypeName, string? c
             var serviceContainer = await VSShell.ServiceExtensions.GetServiceAsync<SVsBrokeredServiceContainer, IBrokeredServiceContainer>(_asyncServiceProvider).ConfigureAwait(false);
             var service = serviceContainer.GetFullAccessServiceBroker();
 
-            var configuration = await TraceConfiguration.CreateTraceConfigurationInstanceAsync(service, cancellationToken).ConfigureAwait(false);
+            var configuration = await TraceConfiguration.CreateTraceConfigurationInstanceAsync(service, ownsServiceBroker: true, cancellationToken).ConfigureAwait(false);
```

Comment:
I also get a diagnosis that this method should use the overload with 'ownServiceBroker' 
From http://index/?leftProject=Microsoft.VisualStudio.Utilities&leftSymbol=zoucrdiplxd9 I feel this should be correct.
But tag @dibarbet to verify : )